### PR TITLE
fix: verify SecretsManager JSON Key

### LINKED ISSUE #532: `ecspresso verify` cannot validate secret value when using JSON keys in AWS Secrets Manager
## summary
Verify command cannot validate secret value when using JSON keys in AWS Secrets Manager.
Returns OK even if it specify a non-existing Key.
Is this intended behavior?

## actual behavior
```
ecspresso verify
2023/03/14 12:42:46 ecs-service/ecs-cluster Starting verify
  TaskDefinition
...
    ContainerDefinition[xxxx]
      Secret TEST_1[arn:aws:secretsmanager:ap-northeast-1:***:secret:my-secret-xxxxxx:::]
      --> [OK]
      Secret TEST_2[arn:aws:secretsmanager:ap-northeast-1:***:secret:my-secret-yyyyyy:NOT_EXIST_FIELD::]
      --> [OK]
      Secret TEST_3[arn:aws:secretsmanager:ap-northeast-1:***:secret:not-exist-secret:::]
      --> [NG]
...
```
## expected behavior
```
ecspresso verify
ecs-service/ecs-cluster Starting verify
  TaskDefinition
...
    ContainerDefinition[xxxx]
      Secret TEST_1[arn:aws:secretsmanager:ap-northeast-1:***:secret:my-secret-xxxxxx:::]
      --> [OK]
      Secret TEST_2[arn:aws:secretsmanager:ap-northeast-1:***:secret:my-secret-yyyyyy:NOT_EXIST_FIELD::]
      --> [NG]
      Secret TEST_3[arn:aws:secretsmanager:ap-northeast-1:***:secret:not-exist-secret:::]
      --> [NG]
...
```

## version
2.0.1

### PR DESCRIPTION
**Pull request by m22r**
Fixed `ecspresso verify` command to verify secretsManager JSON Key

fix #532

### CODE REVIEW

File: verify.go [hunk 1]
```diff
@@ -65,12 +65,24 @@ func (v *verifier) existsSecretValue(ctx context.Context, from string) error {
...
 		}
 		secretArn := strings.Join(part[0:7], ":")
-		_, err := v.secretsmanager.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{
+		res, err := v.secretsmanager.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{
 			SecretId: &secretArn,
 		})
 		if err != nil {
 			return fmt.Errorf("failed to get secret value from %s secret id %s: %w", from, secretArn, err)
 		}
+		key := part[7]
```
Comment (fujiwara) [inline verify.go:77]: `part` slice contains only 7 elements if json key not specified. So this lookup raise a panic (`runtime error: index out of range [7] with length 7`).

```suggestion
		if len(part) < 8 {
			return nil
		}
		key := part[7]
```

**Review by fujiwara — CHANGES_REQUESTED** [review]
@m22r Thank you! Please confirm a review comment.

**Comment by m22r** [comment]
@fujiwara Thank you for your review comment. I had fixed in [3e0e3fd](https://github.com/kayac/ecspresso/pull/533/commits/3e0e3fd41ed4713f065ec20f95bd9e94725085d4)

**Review by fujiwara — APPROVED** [review]

**Comment by fujiwara** [comment]
@m22r Thank you!

**Pull request merged**